web/public/css/dashboard.less

```diff
@@ -1820,3 +1817,27 @@ span.applications-banner {
     13em 22em,
     14em 22em;
 }
+// Classes made to fix padding should eventually be moved for coresponding dierectives
+.padding {
```

Comment:
Should have a more descriptive name. You couldn't use this as padding for a button for example.